Documentation/ApiOverview/RequestLifeCycle/RequestAttributes/CurrentContentObject.rst

```diff
@@ -30,3 +30,5 @@ Example:
      * @var \TYPO3\CMS\Frontend\ContentObject\ContentObjectRenderer $currentContentObject
      */
     $currentContentObject = $request->getAttribute('currentContentObject');
+    //ID of current tt_content record
```

Comment:
```suggestion

    // ID of current tt_content record
```

Reply:
@cweiske Can you please commit the suggestion? In future PRs please allow us to change your adjustments. This makes it easier for us all.